tbclient/documentation.go

```diff
@@ -0,0 +1,35 @@
+package tbclient
+
+import "fmt"
```

Comment:
Documentation is another snowflake service, as it uses the Topology UID as its own UID and only has Get/Create functions.